tgui/packages/tgui/styles/main.scss

```
@@ -54,6 +54,7 @@
 @include meta.load-css('./interfaces/CrewManifest.scss');
 @include meta.load-css('./interfaces/ExperimentConfigure.scss');
 @include meta.load-css('./interfaces/HellishRunes.scss');
+@include meta.load-css('./interfaces/HotkeysHelp.scss');
```

```suggestion
@include meta.load-css('./interfaces/HotKeysHelp.scss');
```
Without this it fails on linux.

---

whoops.